woocommerce/Blocks/Traits/Block_Integration_Trait.php

```diff
@@ -36,7 +36,7 @@ trait Block_Integration_Trait
 	 */
 	public function get_name() {
 
-		return $this->plugin->get_id_dasherized();
+		return $this->plugin->get_id();
```

Comment:
@unfulvio-godaddy looks like Woo's convention is to use underscored names, for example `WC_Gateway_Braintree_Credit_Card_Blocks_Support` sets the name as `braintree_credit_card`